### PR TITLE
Misc. documentation and source comment typo fixes

### PR DESCRIPTION
**Pull request by luzpaz**
Found via `codespell -q 3 -L uint,od,objext,ba,cmo,bae,ans,struc,fo,clos,goin,upto,thru`

### CODE REVIEW

File: CHANGES [hunk 1]
```diff
@@ -238,7 +238,7 @@ Version 4.0.0 (27 Apr 2019)
 	    [PHP] Fix access to already released memory during PHP module
 	    shutdown, which often didn't cause visible problems, but could
 	    result in segmentation faults, bus errors, etc.  Fixes #1170,
-	    reported by Jitka Plesn�kov�.
+	    reported by Jitka Plesn�kov�.
```
Comment (ojwb) [inline CHANGES]: It looks like this patch changes lines with accented characters - are these changes intended?  The github patch view doesn't seem to know the right encoding so I just see `?` replacement characters before and after...

File: CHANGES [hunk 2]
```diff
@@ -11138,7 +11138,7 @@ Version 1.3.26 (October 9, 2005)
               valuewrapper
 
             It is now possible, for example to ignore all methods/classes in a header file, except for a
-            few targetted methods, for example:
+            few targeted methods, for example:
```
Comment (ojwb) [inline CHANGES]: That's not a typo - "targetted" is the UK English spelling.

File: Doc/Manual/Python.html [hunk 3]
```diff
@@ -5498,16 +5498,16 @@ <H3><a name="Python_memory_management_member_variables">32.9.7 Memory management
...
+be added to the <tt>wheel</tt> instance.
 </p>
-    
+
 <p>
 You can do this by adding the reference when the <tt>getWheel()</tt> method
 is called using one of three approaches:
 </p>
-    
+
```
Comment (ojwb) [inline Doc/Manual/Python.html]: It'd be better not to mix whitespace cleanup with other changes.

File: CHANGES (hunk 2 above)
Comment (luzpaz) [reply CHANGES]: Will update to `codespell` dictionary with this feedback. thanks. I'll revert this change.

**Comment by wsfulton** [comment]
Please squash the two commits into one to avoid unnecessary noise. Thanks.

**Comment by ojwb** [comment]
Thanks.
